data/human/lusk.txt

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2023-2024 by Alex Rowan
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "BAM: Transport Lusk"
+	name "Transport Lusk to <planet>"
+	description "Transport entrepreneur Johann Lusk to a business meeting on <destination>."
+	source
+		attributes "deep"
+	destination "Ada"
+#	to offer
+#		random < 10
+	passengers 1
+	on offer
```

Comment:
```suggestion
	description "Transport entrepreneur Johann Lusk to a business meeting on <destination>. He promises to pay you <credits>."
	passengers 1
	"apparent payment" 15000
	source
		attributes "deep"
	destination "Ada"
#	to offer
#		random < 10
	on offer
```
The (unpublished) order of precedence for mission syntax places `passengers` between `description` and `source`. In addition, `apparent payment` can be used to display a payment that isn't representative of the end payment.

Reply:
I mentioned this in another comment as well, but it turns out that `apparent payment` is overridden by the real payment in `on complete`. The code looks ahead and spills the beans about how much the player will eventually be paid. Unfortunate.

Reply:
Somehow this got duplicated. Hmm.

Reply:
Apparent payment should work for the description line.

Reply:
Turns out it did not work for the description line either. `<credits>` rendered as "<credits>" in the game, and `<payment>` rendered as "9,486 credits".

Reply:
That's very weird, it contradicts what I've experienced in the past. I made a mistake putting in `<credits>` anyway.